"Screen.h:290 error: 'SH1106Wire' does not name a type" error when enabling USE_SH1106 in config

**Describe the bug**
When enabling the USE_SH1106 define in configuration.h we get the compiler error:

_src/esp32/../graphics/Screen.h:290:5: error: 'SH1106Wire' does not name a type SH1106Wire dispdev;_


**Solution**
There is no link from Screen.h to configuration.h. Proper #include should be added at line 23 to Screeen.h:

`#include "../configuration.h"
`

--- a/src/graphics/Screen.h
+++ b/src/graphics/Screen.h
@@ -22,6 +22,8 @@ class Screen
 
 #include <OLEDDisplayUi.h>
 
+#include "../configuration.h"
+
 #ifdef USE_SH1106
 #include <SH1106Wire.h>
 #elif defined(USE_ST7567)

